deploy/olm-catalog/test/8.0.0-1/manifests/bamoe-businessautomation-operator.clusterserviceversion.yaml

```diff
@@ -7,18 +7,18 @@ metadata:
     categories: Integration & Delivery
     certified: "true"
     containerImage: registry-proxy.engineering.redhat.com/rh-osbs/bamoe-8-rhpam-rhel8-operator:8.0.0
-    createdAt: "2022-07-22 11:26:38"
+    createdAt: "2022-07-25 15:01:51"
     description: Deploys and manages IBM Business Automation Manager Open Editions
       environment.
     operators.openshift.io/infrastructure-features: '["Disconnected"]'
     repository: https://github.com/kiegroup/kie-cloud-operator
     support: IBM
     tectonic-visibility: ocs
   labels:
-    operator-businessautomation: "true"
+    operator-bamoe-businessautomation: "true"
```

Comment:
Is it right operator-bamoe-businessautomation instead of bamoe-operator-businessautomation

Reply:
I think so, not sure for what these labels are used for but I kept the same naming thanbefore.